storybook/src/styles/docs.css

```diff
@@ -165,9 +165,18 @@
   border-inline-start-style: solid;
 }
 
+[class*="ams-docs-token-example--color"] {
+  forced-color-adjust: none;
```

Comment:
It might be cool if this works. ie. You could see what forced colors do to the swatches.